environments/base/manifests/loki-querier-deployment.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: apps/v1
-kind: Deployment
```

Comment:
In general, we always try to have the filename match the k8s resource type, so if the loki querier is a statefulset, then the file should be called `loki-querier-statefulset.yaml`